tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -84,6 +84,11 @@ SPK_ID=6,SPEAKER_SSP_DAI_INDEX=0,HEADSET_CODEC_NAME=SSP2-Codec,SPEAKER_CODEC_NAM
 BT_NAME=SSP1-BT,BT_INDEX=1,BT_ID=7,BT_PCM_NAME=Bluetooth,INCLUDE_ECHO_REF=true,USE_CHAIN_DMA=true,\
 DEEPBUFFER_D0I3_COMPATIBLE=true,GOOGLE_RTC_AEC_SUPPORT=1,DEEP_BUF_SPK=true,PLAYBACK_PIPELINE_SRC=dts"
 
+#ES83x6+HDMI-IN
+"cavs-es83x6\;sof-mtl-es83x6-ssp1-hdmi-ssp02\;PLATFORM=mtl,PREPROCESS_PLUGINS=nhlt,\
+NHLT_BIN=nhlt-sof-mtl-es83x6-ssp1-hdmi-ssp02.bin,HEADSET_SSP_DAI_INDEX=1,\
+HEADSET_CODEC_NAME=SSP1-Codec,HDMI_IN_CAPTURE=true,USE_CHAIN_DMA=true"
```

Comment:
rebase needed, this is HDMI_USE_CHAIN_DMA now.

Reply:
Already rebased please check the latest.